scripts/exregional_make_lbcs.sh

```diff
@@ -82,7 +82,7 @@ fi
 #-----------------------------------------------------------------------
 #
 if [ $RUN_ENVIR = "nco" ]; then
-    extrn_mdl_staging_dir="${COMINext}${SLASH_ENSMEM_SUBDIR}"
+    extrn_mdl_staging_dir="${DATAROOT}/get_extrn_lbcs.${share_pid}${SLASH_ENSEM_SUBDIR}"
```

Comment:
The modification should have been `${SLASH_ENSMEM_SUBDIR}`, not `${SLASH_ENSEM_SUBDIR}`.  I'm very sorry about that.

```suggestion
    extrn_mdl_staging_dir="${DATAROOT}/get_extrn_lbcs.${share_pid}${SLASH_ENSMEM_SUBDIR}"
```

Reply:
Updated!